server/filesystem/compress.go

```diff
@@ -29,6 +29,11 @@ import (
 // and the compressed file will be placed at that location named
 // `archive-{date}.tar.gz`.
 func (fs *Filesystem) CompressFiles(dir string, paths []string) (ufs.FileInfo, error) {
+	for _, file := range paths {
+		if err := fs.IsIgnored(path.Join(dir, file)); err != nil {
+			return nil, err
```

Comment:
Can we just skip adding the file to the archive instead of throwing an error?

Reply:
I added a prity simple fix to filter them first out of the paths